heroic-core/pom.xml

```diff
@@ -149,9 +157,87 @@
     <dependency>
       <groupId>com.google.protobuf</groupId>
       <artifactId>protobuf-java</artifactId>
-      <version>3.5.1</version>
+      <version>3.6.1</version>
+    </dependency>
+
+    <!-- opencensus dependencies -->
+    <dependency>
+      <groupId>io.opencensus</groupId>
+      <artifactId>opencensus-api</artifactId>
+      <version>${opencensus.version}</version>
```

Comment:
you can get rid of all these versions since they are declared already in the parent pom.